packages/flutter/lib/src/material/drawer.dart

```diff
@@ -264,10 +298,28 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   final ColorTween _color = new ColorTween(begin: Colors.transparent, end: Colors.black54);
   final GlobalKey _gestureDetectorKey = new GlobalKey();
 
+  AlignmentDirectional get _drawerStartAlignment {
```

Comment:
The term "start" here is a bit confusing because "start" and "end" are possible alignment values themselves.  For example, it's confusing that a "start alignment" would return "centerEnd".

Maybe `_drawerOuterAlignment` for this one and `_drawerInnerAlignment` for the one below?

Reply:
Fixed